src/Infernus101/KitUI/Main.php

```diff
@@ -41,7 +41,13 @@ public function onEnable(){
       $this->language = new LangManager($this);
       $this->getServer()->getPluginManager()->registerEvents(new PlayerEvents($this), $this);
       $this->getServer()->getScheduler()->scheduleDelayedRepeatingTask(new CoolDownTask($this), 1200, 1200);
-      $this->piggyEnchants = $this->getServer()->getPluginManager()->getPlugin("PiggyCustomEnchants");
+        if ($this->getServer()->getPluginManager()->getPlugin("EconomyAPI") === null) {
+            $this->getLogger()->critical('Please install EconomyAPI ... The plugin is stopped ...');
+            $this->getServer()->getPluginManager()->disablePlugin($this);
```

Comment:
You can just use
`$this->setEnabled(false);`
Also you may use _**return**_ to interrupt enabling

Reply:
Or not, _**disablePlugin()**_ would be better. My bad